tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -4,11 +4,29 @@
 
 namespace OpenTelemetry\Test\Unit\Instrumentation\AwsSdk;
 
+use OpenTelemetry\API\Trace\TracerInterface;
+use OpenTelemetry\Aws\Xray\IdGenerator;
+use OpenTelemetry\Aws\Xray\Propagator;
 use OpenTelemetry\Instrumentation\AwsSdk\AwsSdkInstrumentation;
+use OpenTelemetry\SDK\Trace\SpanExporter\ConsoleSpanExporter;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
 use PHPUnit\Framework\TestCase;
 
 class AwsSdkInstrumentationTest extends TestCase
 {
+    private TracerProvider $tracerProvider;
+    private AwsSdkInstrumentation $awsSdkInstrumentation;
+
+    protected function setUp(): void
+    {
+        $spanProcessor = new SimpleSpanProcessor(new ConsoleSpanExporter());
```

Comment:
phpunit will probably be unhappy if this exporter ever actually exported anything (tests shouldn't create output).
Because this is a unit test, and your code should depend on interfaces, you can use mock objects: `$this->tracerProvider = $this->createMock(TracerProvider::class)`. The only "real" class that you should use is the class(es) you're testing: AwsSdkInstrumentation